waspc/src/Parser/Action.hs

```diff
@@ -0,0 +1,27 @@
+module Parser.Action
+    ( action
+    ) where
+
+import Text.Parsec.String (Parser)
+
+import qualified Wasp.Action as Action
+
+import qualified Parser.JsCode
+import qualified Lexer as L
+
+
+action :: Parser Action.Action
+action = do
+    -- TODO: Code below, for smth<entity> ... should be reused, there is already
+    --   a function in Parser.Common for smth similar but it was assuming closure {}
```

Comment:
yes we could make a more general function which does not assume closure and then use it for both. But ok if you don't want to do that now.

Reply:
Ok, I did as you suggested!